ui/themes/app.js

```diff
@@ -117,7 +117,7 @@ export var darkTheme = createTheme({
       disableButtonBg: '#666666',
       disableButton: '#9d9d9d',
       listItemHover: 'rgb(0, 187, 166, 0.5)',
-      white: 'rgb(255, 255, 255, 0.1)',
+      white: '#202020',
```

Comment:
// FYI @ritiksaxena124

Reply:
@senali-d have you tested this theme change in MeshMap too?

Reply:
This color define specifically for environments in my previous PR. I think this was not use anywhere other than this page.

Reply:
#9647

Reply:
Ohh make sense thanks for confirming